test/metric_value_benchmark/agent_resources/emf_counter.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "EMF structured log schema",
+  "description": "JSON schema for the CloudWatch agent structured log",
+  "type": "object",
+  "properties": {
+    "_aws": {
+      "$ref": "#/definitions/emfDefinition"
+    },
+    "Type": {
+      "type": "string"
+    },
+    "EMFCounter": {
+      "type": "integer"
+    },
+    "InstanceId": {
+      "type": "string"
+    }
+  },
+  "required": [
+    "_aws",
+    "Type",
+    "EMFCounter",
+    "InstanceId"
+  ],
+  "additionalProperties": true,
+  "definitions": {
+    "emfDefinition": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "Timestamp": {},
```

Comment:
Why would this be an empty type though?

Reply:
I don't think that it makes much difference to validate the exact types here or not. If you feel strongly about it I can change it.